example/rcnn/rcnn/config.py

```diff
@@ -134,7 +134,7 @@
 network.resnet.RPN_FEAT_STRIDE = 16
 network.resnet.RCNN_FEAT_STRIDE = 16
 network.resnet.FIXED_PARAMS = ['conv0', 'stage1', 'gamma', 'beta']
-network.resnet.FIXED_PARAMS_SHARED = ['conv0', 'stage1', 'stage2', 'stage3', 'gamma', 'beta']
+network.resnet.FIXED_PARAMS_SHARED = ['conv0', 'stage1', 'stage2', 'stage3', 'bn0', 'bn_data']
```

Comment:
Oh. Sorry. I tried alternative training which is not mentioned here.
It should be 'gamma' and 'beta'. I'll fix it.